corehq/apps/couch_sql_migration/diff.py

```diff
@@ -109,6 +108,8 @@
         # defaulted on SQL
         Ignore('missing', ('indices', '[*]', 'relationship'), old=MISSING, new='child'),
 
+        Ignore(path=('actions', '[*]')),
```

Comment:
I think this needs to change to

```suggestion
        Ignore(check=is_case_actions),
```

and this will need to be added below

```py
def is_case_actions(old_obj, new_obj, rule, diff):
    return diff.path[0] == "actions"
```